dev-client/src/translations/en.json

```diff
@@ -507,6 +507,10 @@
       "photos": "Photos",
       "notes": "Notes"
     },
+    "collection_method_description": {
```

Comment:
Is it intended that only these methods have descriptions? I'm seeing a bunch of warning spew in my logs about missing i18n keys for the other methods. I feel that we should avoid those warnings by some means.

Reply:
Yes, only some methods have descriptions. I've added a check to avoid the warnings.